docs/wallet-integration-guide/src/exchange-integration/extensions.rst

```diff
@@ -6,16 +6,94 @@ Integration Extensions
+`integration test
+<https://github.com/digital-asset/canton/blob/3c9ac9891c03cb06303736d7224bcc01dbd50084/community/app/src/test/scala/com/digitalasset/canton/integration/tests/jsonapi/ExternalPartyLedgerApiOnboardingTest.scala#L183>`_. Note
+in particular that you must sign the ``PartyToParticipant`` mapping
+not just by your party's key but also by all confirming
+participants. This is accomplished through the
+``participant2.topology.transactions.authorize`` step in the test.
+
+DAR Management
+^^^^^^^^^^^^^^
+
+Any DAR that you upload, both as part of the initial setup but also
+whenever you upload newer versions to upgrade an existing package,
+must be uploaded to all confirming nodes of the party.
+
+Reading Data and Submitting Transactions
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You can use any of the confirming nodes for reading data of a party
+hosted on them. However, offsets are not comparable across nodes so it
+is recommended that to run Tx History Ingestion against the same node
+under normal operations. If you do need to switch nodes, you can do so
+following the same procedure used for `restoring a validator from a
+backup <validator_backup_restore>`_ to resynchronize Tx History
+Ingestion against the offsets of the new node.
+
+Preparation and execution of transactions can also be done against any
+of the confirming nodes of the party. However, `Command Deduplication
+<https://docs.digitalasset.com/build/3.3/sdlc-howtos/applications/develop/command-deduplication.html>`_
+is only performed by the executing node so if you submit across nodes you cannot rely on it.
+
+.. TODO:: Link to recommended deduplication strategy https://github.com/hyperledger-labs/splice-wallet-kernel/issues/423
+
+Changing the set of Confirming Nodes
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+It is highly recommended to setup your desired set of confirming nodes
+when initially setting up the party as changes to the set of
+confirming nodes are tricky. Adding and removing nodes is possible but
+it is a fragile procedure that is not recommended in production at
+this point. This will be improved in future releases.
+
+If you do find yourself in a situation where you don't have any other
+option than changing the set of confirming nodes, refer to the `Canton documentation <https://docs.digitalasset.com/operate/3.3/howtos/operate/parties/party_replication.html#offline-party-replication>`_ for detailed information on the limitations. At a high level, the limitations are as follows:
+
+Removing nodes is possible by submitting a new ``PartyToParticipant`` mapping but can leave
+the nodes hosting the party in a broken state so make sure that those
+nodes were not used for other purposes.
+
+Adding new nodes does not just require changing the
+``PartyToParticipant`` mapping but it also requires a manual state
+transfer of all active contracts to new confirming nodes. This is an
+involved procedure with high potential for issues.
```

Comment:
This reads scary. I'd suggest to only explain the option based on allocating new treasuryParties; and the option to remove hosting nodes.

How broken is the removal?

Reply:
> This reads scary. I'd suggest to only explain the option based on allocating new treasuryParties; and the option to remove hosting nodes.

good idea, will rephrase.

> How broken is the removal?

Not fully sure, checking with Canton

Reply:
> Not fully sure, checking with Canton

TLDR: Badly tested, not thoroughly analyzed and some known issues but those are around reassignments. So I'll stick to the cautious phrasing here.